lib/player.js

```diff
@@ -2758,7 +2742,7 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       getPlayRate: () => this.playRateController_ ?
         this.playRateController_.getRealRate() : 0,
       getPresentationTime: () => this.playhead_.getTime(),
-      getPresentationLatency: () => this.getPresentationLatency(),
+      getEncoderLatency: () => this.getEncoderLatency(),
```

Comment:
just keep getPresentationLatency, EncoderLatency means the latency delta caused by encoder

Reply:
Yes I'm currently removing it